website/content/docs/platform/k8s/injector/annotations.mdx

```diff
@@ -163,9 +166,19 @@ Vault annotations change how the Vault Agent containers communicate with Vault.
 example, Vault's address, TLS certificates to use, client parameters such as timeouts,
 etc.
 
-- `vault.hashicorp.com/auth-path` - configures the auth path for the Kubernetes
+- `vault.hashicorp.com/auth-config` - configures additional parameters for the configured 
+  authentication method. The name of the config is any unique string after 
+  `vault.hashicorp.com/auth-config-`, such as `vault.hashicorp.com/auth-config-role-id-file-path`. 
```

Comment:
Is the subset of auth methods small enough for this annotation-based config that we can simply list them here?

Reply:
It's all auto-auth methods. I'll just make a link to auto-auth doc.